### PR TITLE
Adapting a TTS API for the GSVI (GPT-SoVits Inference) Project

### PR DESCRIPTION
**Pull request by X-T-E-R**
GSVI (GPT-Sovits Inference) is an inference enhancement project based on GPT-Sovits, allowing you to run an API interface locally, offering emotion-rich speech-to-text and convenient model management features.

https://github.com/X-T-E-R/GPT-SoVITS-Inference
https://github.com/RVC-Boss/GPT-SoVITS
https://github.com/X-T-E-R/TTS-for-GPT-soVITS

image

### CODE REVIEW

File: public/scripts/extensions/tts/gsvi.js [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+        top_p: 0.85,
+        temperature: 0.75,
+        batch_size: 10,
+
+        stream: false,
+        stream_chunk_size: 100,
+    };
+
+
+    // 新增获取角色和情绪的方法
```
Comment (Cohee1207) [inline public/scripts/extensions/tts/gsvi.js]: Please use English in the comments and strings. Maintainers are not fluent in other languages

**Comment by Cohee1207** [comment]
I couldn't test it because installing GPT-SoVits Inference requires registration on some Chinese Git host unless I'm missing something.

**Comment by X-T-E-R** [comment]
> I couldn't test it because installing GPT-SoVits Inference requires registration on some Chinese Git host unless I'm missing something.

Sorry I am testing the English Version, you could go to GSVI page now : https://github.com/X-T-E-R/GPT-SoVITS-Inference

**Comment by X-T-E-R** [comment]
https://huggingface.co/XTer123/GSVI_prezip/tree/main
Or just using the prezip for Windows !

**Comment by Pevernow** [comment]
What is the situation now?

**Comment by Cohee1207** [comment]
I can merge after 1.12.0 release.

**Comment by frammiie** [comment]
Just tested this, fully works! Amazing, thank you for your hard work @X-T-E-R ! 🤗 谢谢 !

**Pull request merged**